tools/update_to_relative_path/MakeArchiveLocationRelative.pl

```diff
@@ -0,0 +1,218 @@
+#! /usr/bin/perl
```

Comment:
I still think we should not have this very specific sub-directory (as otherwise, every script will be in its own directory).
I would suggest maybe moving to the `tools' for now, then maybe in another PR, re-organize the scripts within the tools/ once we know more how to logically split them